src/main/java/com/booleanuk/extension/Extension.java

```diff
@@ -70,8 +73,11 @@ public StringBuilder four() {
         // 1. Using the sb variable above, add "Hello, world!" to the StringBuilder
         // 2. After adding the message, replace the word "world" with the word "Java"
         // WRITE YOUR CODE BETWEEN THIS LINE...
-
-
```

Comment:
Works, but doesn't need to be so complex. Generally speaking, if it can be made more readable, that is the best approach.